src/test/java/com/sampleweb/journey/LargeAppliancesTests.java

```diff
@@ -0,0 +1,56 @@
+package com.sampleweb.journey;
+
+import com.sampleweb.BaseTest;
+import com.sampleweb.framework.PageNavigator;
+import com.sampleweb.pages.HomePage;
+import com.sampleweb.pages.LargeAppliancesPage;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.core.Is.is;
+
+public class LargeAppliancesTests extends BaseTest {
+    private PageNavigator navigator = new PageNavigator();
+    private LargeAppliancesPage largeAppliancesPage;
+
```

Comment:
If you add the `@BeforeMethod` annotation you can remove the `setUp()` function call from all the other test functions

Reply:
Fixed :)

Reply:
He seemed to wanted us to do it that way, so that we dont use the path in the class.